README.rst

```diff
@@ -1,4 +1,4 @@
 cdist manifestation
 ===================
 
-A library which helps you with calling cdist types from manifests written in Python.
+A library for calling cdist types from manifests written in Python.
```

Comment:
What about "A helper for writing cdist manifests in Python"?

Reply:
That's better, changed.